.github/_README.md

```diff
@@ -0,0 +1,216 @@
+<!-- GitHub will publish this readme on the main repo page if the name is `README.md` so we've added the leading underscore to prevent this -->
```

Comment:
I think `ACTIONS_README` is great. I like having this as part of the repo because it gives clear explanations for anyone who might not be familiar with GH Actions